src/containers/scanner/redux/saga.js

```diff
@@ -1,19 +1,25 @@
 // @flow
-import { put, takeEvery, delay } from "redux-saga/effects";
+import { put, takeEvery, call } from "redux-saga/effects";
 import type { Saga } from "redux-saga";
 import { loadScanPending, loadScanSuccess, loadScanFailure } from "./actions";
-import { LOAD_SCANNER } from "./constants";
+import * as constants from "./constants";
+import { getUserByCarNumber } from "../../../services/api";
+// $FlowFixMe
+import RNTextDetector from "react-native-text-detector";
+import * as NavigationService from "../../../services/NavigationService";
```

Comment:
would be good to use an absolute path, I've created a task for it  #forthefuture :)